### ISSUE TITLE
Moon phase?

### ISSUE THREAD

**Issue by theojurriens** [migueln/PySQM#3]
Is it possible the write the Moonphase in the Statistics_SQM file. Could be usefull to check data for moonless nights?

Thanks

**Comment by mireianievas** [comment]
Yes, that's definitely possible. I will try to look into it as soon as I have time.
